django/contrib/admin/options.py

```diff
@@ -101,6 +101,11 @@ class IncorrectLookupParameters(Exception):
     pass
 
 
+@cache
+def default_action_name(name):
+    return capfirst(name.replace("_", " "))
```

Comment:
I opted for a module-level function rather than another `staticmethod` cluttering `ModelAdmin`. Perhaps `get_action_description` could even be turned into a plain function, but it would break users who’ve overridden it...

Reply:
Is it really worth caching? Is it worth increasing memory usage for projects with many actions? I don't see this as a hot-path that it's worth such a minor optimization.

Reply:
The cache is just a string-to-string mapping, with the keys already in memory as `func.__name__`. I was trying to guard against the case where a description-less action is `AdminSite.actions` and its action name would be computed for each `ModelAdmin` during checks.

But sure, we can drop it if you still desire.

Reply:
> But sure, we can drop it if you still desire.

If you don't mind :wink:

Reply:
Done. Profiling shows the same speed in my test project, although I don’t think it has many actions without descriptions.